resolwe/flow/views/annotations.py

```diff
@@ -108,8 +128,27 @@ def create(self, request, *args, **kwargs):
             entity.has_permission(Permission.EDIT, request.user) for entity in entities
         ):
             raise exceptions.PermissionDenied()
-        self.perform_create(serializer)
+
+        # Create annotation values.
```

Comment:
Why don't you override the `perform_create` method to keep the structure consistent?

Reply:
Refactored.